content/Arduino Cloud/General Cloud/Configure-the-network-credentials-of-a-Thing.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Configure the network credentials of a Thing"
+id: 14416141314332
+---
+
+In this article:
+
+- [Configuring a Thing without preset network credentials](#without_preset)
+- [Configuring a Thing with preset network credentials](#preset)
+
+## Configuring a Thing without preset network credentials {#without_preset}
+
+1. Go to the [Things tab](https://app.arduino.cc/things) and click the "+Thing" button in the top right corner.
+
+    ![Thing button in the top right corner](img/thing_button.png)
+
+1. After creating a Thing, select your device by clicking the "Select Device" button in the "Associated Device" section.
+
+    ![associated device section with select device button](img/associated_device.png)
+
+1. After selecting your device, you can enter your network credentials.
+
+    ![network credential section with configure button](img/Network_add.png)
+
+    Click the "Configure" button to enter your network credentials.
+
+    - Wi-Fi: Enter your Wi-Fi name (SSID) and password (encryption key).
+    - Arduino SIM: Use these credentials.
+    - Arduino with LoRa®: The credentials are added automatically.
+    - ESP32 and ESP8266: Enter the secret key that was provided when adding the device.
+
+## Configuring a Thing with preset network credentials {#preset}
+
+1. Go to the [Things tab](https://app.arduino.cc/things) and select the Thing that needs its network credentials updated.
+
+1. In your Thing's settings, find the Network section and click the button labeled "Change" to edit the network credentials.
```

Comment:
```suggestion
1. Go to the **Network Credentials** section and click the **Change** button.
```
Since when opening a Thing the "Setup" tap is shown by default, we can make the description more direct to avoid confusion or having users look for the settings somewhere else in the page.